### PR TITLE
Define custom network throttle capabilities

### PR DESCRIPTION
**Pull request by christian-bromann**
Currently we allow to throttle the network based on a set of predefined profiles we provide. It would be great if users can also define custom throttle capabilities in case non of the profiles fits their needs.

### CODE REVIEW

**Comment by christian-bromann** [comment]
This patch can be tested using `speedo@beta`

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ _default:_ `false`
 
 ##### `--throttleNetwork`
 
-Throttle network speed for your test (e.g. "Good 3G"). Available throttling profiles are: `offline`, `GPRS`, `Regular 2G`, `Good 2G`, `Regular 3G`, `Good 3G`, `Regular 4G`, `DSL`, `Wifi`, `online`.
+Throttle network speed for your test (e.g. "Good 3G"). Available throttling profiles are: `offline`, `GPRS`, `Regular 2G`, `Good 2G`, `Regular 3G`, `Good 3G`, `Regular 4G`, `DSL`, `Wifi`, `online`. You can also define your custom network profile by providing 3 with comma separated numbers for download (_kb/s_), upload (_kb/s_) and latency (_ms_), e.g. `1000,500,40`.
```
Comment (maciejurbanskisaucelabs) [inline README.md]: > 3 with comma separated

just `3 comma separated` ?

Comment (christian-bromann) [reply README.md]: What other information do you expect? According to the [Sauce wiki](https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands#CustomSauceLabsWebDriverExtensionsforNetworkandLogCommands-ThrottleNetworkCapabilities) only upload, download and latency is supported.

**Comment by maciejurbanskisaucelabs** [comment]
Do You think it would be worth to pass these throttle parameters to the backend, so we can use these data when comparing tests?

**Comment by christian-bromann** [comment]
> Do You think it would be worth to pass these throttle parameters to the backend, so we can use these data when comparing tests?

We can notify the backend on environment changes like network or CPU throttling but it would be the job of our internal driver, not Speedo.

File: README.md (hunk 1 above)
Comment (maciejurbanskisaucelabs) [reply README.md]: I think it should be written `3 comma separated` instead of `3 with comma separated`

**Pull request merged**